.ci/README.md

```diff
@@ -0,0 +1,15 @@
+# Contiguous Integration (CI) Test for Board Support Pack (BSP)
```

Comment:
Typo: Continuous  integration

Reply:
Typos "Contiguous Integration" -> "Continuous Integration" were solved in separate commit.